src/reconcile/rec_write.c

```diff
@@ -2627,7 +2627,7 @@ __rec_write_wrapup(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_PAGE *page)
     if (WT_TIME_AGGREGATE_HAS_STOP(&stop_ta)) {
         WT_RET(__wt_calloc_one(session, &stop_tap));
         WT_TIME_AGGREGATE_COPY(stop_tap, &stop_ta);
-        WT_PUBLISH(mod->stop_ta, stop_tap);
+        WT_WRITE_ONCE(mod->stop_ta, stop_tap);
```

Comment:
I think this need to be a release write in terms of c memory model. It is custom-release ordering.

Reply:
Fixed as suggesed.